crystalfontz/dbus/map.py

```diff
@@ -0,0 +1,159 @@
+from typing import ClassVar, Optional, Protocol, Self, Tuple, Type, Union
+
+from crystalfontz.config import Config
+from crystalfontz.dbus.config import ConfigStruct
+from crystalfontz.device import Device
+from crystalfontz.response import Pong, Versions
+
+
+class TypeProtocol(Protocol):
+    t: ClassVar[str]
+
+
+def t(*args: Union[str, Type[TypeProtocol]]) -> str:
```

Comment:
I considered doing something similar for dumping and loading structs. Unfortunately, it's not something that's easily made type safe.